web/data/config.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Super Metroid Practice Hack",
-    "version": "2.2.8",
+    "version": "2.2.9",
```

Comment:
If we do all of these changes at once, we might as well go to 2.3.0, especially since we should decrement the artificial lag for total's changes.